app/models/concerns/cable_ready/updatable.rb

```diff
@@ -228,7 +228,7 @@ def broadcast_updates(model_class, options)
       end
 
       def debounce_time
-        @debounce_time ||= 0.seconds
+        @debounce_time ||= CableReady.config.updatable_debounce_time
```

Comment:
Should we coerce the value type with a `to_f` here?

Reply:
hmmm actually that's done in line 221 where it makes more sense?

Reply:
But 215 is expecting a numeric `if debounce_time > 0.seconds`. Just trying to guard against people doing silly stuff.

Reply:
I guess setting it to `nil` could be considered a valid use case depending on the individual? 🤷🏻‍♂️

Reply:
gotcha 👍